src/test/java/org/broadinstitute/hellbender/MainTest.java

```diff
@@ -94,4 +94,17 @@ public void testMainErrorWithoutStackTrace() {
             System.setSecurityManager(backup);
         }
     }
+    @Test(singleThreaded = true)
+    public void testNonZeroPicardReturnValue() {
+        final SecurityManager backup = System.getSecurityManager();
+        try {
+            System.setSecurityManager(new ThrowOnExitSecurityManager());
+            new Main().mainEntry(new String[]{"ExtractSequences"});
```

Comment:
Check the return value of `mainEntry()` to be sure that it's equal to the expected value

Reply:
Done.